### ISSUE TITLE
Sitemap/ URLs inconsistency

### ISSUE THREAD

**Issue by lion152** [scandipwa/base-theme#925]
**Description**:

There is an inconsistency between URLs in sitemap and on-site URLs

URLs in sitemap by default are missing /category/ and /product/ part while can include .html part.

**Expected behavior**:

Remove .html part from URLs
Add /product/ for products and /category/ for categories the same way as you can find a page on the website

Expected outcome:
URLs in sitemap and on the website have the same structure. If there is /category/ part on website's URLs - then it should be also implemented into the sitemap

How to test it:
1) Generate sitemap
1.1. Go to BE  -> Marketing - Site map
1.2. Press "Add a sitemap"
1.3. Filename- "sitemap.xml"
1.4. path - "pub/media/
2) Open generated sitemap
3) Randomly open 5-10 URLs in it. They all should be working and returning 200 status code

**Comment by MaksimGritcenko** [comment]
Looks like the issue doesn`t exist anymore! I`ve created sitemaps for a local project, [ScandiPWA demo](https://demo.scandipwa.com/) and one of the projects [Gocciani](https://gocciani-jsq-1593419306.scandipwa.cloud/). All the created sitemaps appropriate to the existing pages! .html parts are also shown in pages paths, therefore, must be in sitemaps. /category, /product and analogous pieces of URLs don`t exist for ScandiPWA projects.

     But, for Gocciani, in a sitemap, I found some URLs which return 404. After all, it turned out, that the pages for these products don`t exist on FE, while the products are on BE. therefore it`s not a sitemap issue
![Screenshot from 2020-10-01 13-47-06](https://user-images.githubusercontent.com/59978452/94801353-957c8e80-03ee-11eb-9eba-ae1691f4519c.png)
![Screenshot from 2020-10-01 13-47-18](https://user-images.githubusercontent.com/59978452/94801357-96152500-03ee-11eb-8fc6-bee49f2bca5a.png)

!

**Comment by MaksimGritcenko** [comment]
Looks like the issue doesn`t exist anymore! I`ve created sitemaps for a local project, [ScandiPWA demo](https://demo.scandipwa.com/) and one of the projects [Gocciani](https://gocciani-jsq-1593419306.scandipwa.cloud/). All the created sitemaps appropriate to the existing pages! .html parts are also shown in pages paths, therefore, must be in sitemaps. /category, /product and analogous pieces of URLs don`t exist for ScandiPWA projects.

     But, for Gocciani, in a sitemap, I found some URLs which return 404. After all, it turned out, that the pages for these products don`t exist on FE, while the products are on BE. therefore it`s not a sitemap issue!

**Comment by MaksimGritcenko** [comment]
Looks like the issue doesn't exist anymore! I've created sitemaps for a local project, [ScandiPWA demo](https://demo.scandipwa.com/) and one of the projects [Gocciani](https://gocciani-jsq-1593419306.scandipwa.cloud/). All the created sitemaps appropriate to the existing pages! .html parts are also shown in pages paths, therefore, must be in sitemaps. /category, /product and analogous pieces of URLs don`t exist for ScandiPWA projects.

     But, for Gocciani, in a sitemap, I found some URLs which return 404. After all, it turned out, that the pages for these products don`t exist on FE, while the products are on BE. therefore it`s not a sitemap issue!

**Comment by MaksimGritcenko** [comment]
![Screenshot from 2020-10-01 13-47-06](https://user-images.githubusercontent.com/59978452/94802746-dd041a00-03f0-11eb-82b7-772b9e39a6db.png)
![Screenshot from 2020-10-01 13-47-18](https://user-images.githubusercontent.com/59978452/94802750-dd9cb080-03f0-11eb-9200-7c215293fc18.png)

**Comment by MaksimGritcenko** [comment]
Looks like the issue doesn't exist anymore! 

I've created sitemaps for a local project, [ScandiPWA-demo](https://demo.scandipwa.com/) and one of the projects [Gocciani](https://gocciani-jsq-1593419306.scandipwa.cloud/). All the created sitemaps appropriate to the existing pages! .html parts are also shown in pages paths, therefore, must be in sitemaps. /category, /product and analogous pieces of URLs don't exist for ScandiPWA projects.

But, for Gocciani, in a sitemap, I found some URLs which return 404. After all, it turned out, that the pages for these products don't exist on FE, while the products are on BE. therefore it's not a sitemap issue!
